includes/definitions/discovery/dell-net.yaml

```diff
@@ -0,0 +1,80 @@
+mib: DELL-NETWORKING-CHASSIS-MIB:DELL-NETWORKING-IF-EXTENSION-MIB
+modules:
+    processors:
+        data:
+            -
+                snmp_flags: ['-ObsQ']
+                oid: dellNetCpuUtilTable
+                value: dellNetCpuUtil5Sec
+                num_oid: '.1.3.6.1.4.1.6027.3.26.1.4.4.1.1.{{ $index }}'
+                descr: '5s Utilization'
+                index: 'a_dellNetCpuUtil5Sec.{{ $index }}'
```

Comment:
Probably not so necessary to poll the 5s utilization. It will be high (because it will mostly count the SNMP activity from LibreNMS).

Reply:
has been destroyed